test/testenv/verificationutils.go

```diff
@@ -423,3 +423,27 @@ func VerifyCPULimits(deployment *Deployment, ns string, podName string, expected
 		return result
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(true))
 }
+
+// VerifySecretObjectUpdate Check whether the secret object info is pushed to other pods
+func VerifySecretObjectUpdate(deployment *Deployment, testenvInstance *TestEnv, verificationPods []string, secretKey string, modifiedKey string) bool {
+	for _, pod := range verificationPods {
+		key := GetSecretKey(deployment, testenvInstance.GetName(), secretKey, pod)
+		if key != modifiedKey {
+			testenvInstance.Log.Info("Key not updated on pod", "Pod", pod)
```

Comment:
Add more data to Log as explained below.

Reply:
added